editoast/src/schema/operation/create.rs

```diff
@@ -27,6 +28,63 @@ pub enum RailjsonObject {
     Catenary { railjson: Catenary },
 }
 
+impl Cache for RailjsonObject {
```

Comment:
Yes it can be removed, done and replaced by https://github.com/osrd-project/osrd/pull/5924/commits/a84101ff527c5628dd3a48fad13eced634b0d5b8#diff-fb61e16eab3011697c20a8dd2207aa73d56c484f64a982863c08577181ffc5a7R54-R73.